src/common/document_errors.rs

```diff
@@ -67,6 +67,7 @@ lazy_static! {
             (NitroCliErrorEnum::LoggerError, "E56"),
             (NitroCliErrorEnum::HasherError, "E57"),
             (NitroCliErrorEnum::EnclaveNamingError, "E58"),
+            (NitroCliErrorEnum::SignatureCheckerError, "E59"),
```

Comment:
Could use here: "EIFSignatureCheckerError", to be more specific which signature.